kuma/static/js/wiki.js

```diff
@@ -191,11 +191,21 @@
         Syntax highlighting scripts
     */
     if($('article pre').length && ('querySelectorAll' in doc)) (function() {
-        var syntaxScript = doc.createElement('script');
-        syntaxScript.setAttribute('data-manual', '');
-        syntaxScript.async = 'true';
-        syntaxScript.src = mdn.staticPath + 'js/syntax-prism-min.js?build=' + mdn.build;
-        doc.body.appendChild(syntaxScript);
+        var bundle = 'syntax-prism';
+        if (window.mdn && mdn.assets.js && mdn.assets.js.hasOwnProperty(bundle)) {
+            mdn.assets.js[bundle].forEach(function(bundle_url, index, array) {
+                var syntaxScript = doc.createElement('script');
+                syntaxScript.setAttribute('data-manual', '');
+                syntaxScript.src = bundle_url;
+                if (index === 0 && array.length > 1) {
+                    /* In development, slow load first script */
```

Comment:
On further testing, ``async = 'false'`` is a truthy value too, so this code does nothing.  Unless we use something stronger (like [require.js](http://www.requirejs.org)), we can't guarentee load order.  Luckily, a refresh usually fixes it.  Changing the comment to an apology.  See [SO 9711160](http://stackoverflow.com/questions/9711160/jquery-load-scripts-in-order) for more ideas.